### DISCUSSION TITLE
Add product documentation

### DISCUSSION THREAD

**Discussion by aam1r** [outline/outline#2926] [Ideas]
We have been using Outline for ~1 year internally. Every new team member (whether FT or contractor) is onboarded onto it so we can update documentation. However, there's usually some confusion on how to use the product and what is supported.

Is there available Product Documentation for Outline? If not, it would be very useful to have a guide that mentions available features and how to use them.

Thank you 🙌

**Comment by almereyda** [comment]
This discussion continued in #3287, which anticipates the existence of a user guide at some point later this year.
